src/assets/config/version.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
how was this file generated and is it ok to commit it?

Reply:
and is it ok to commit it? Why it wasn't committed before?

Reply:
This is a generated file and it shouldn't be committed. Its also included in `modules/web/.gitignore` and I'm not sure how it got into `src/assets/config` because that directory has been moved to `modules/web/src/assets/config`.